### PR TITLE
Adds gov content

### LINKED ISSUE #151: Add governance mechanism diagram, AQB diagram, and make the equation readable
When I go to learn about governance here http://wiki.polkadot.network/en/latest/polkadot/learn/governance/

It shows me how AQB works as follows:

> The formulae for a positive turnout bias ($B^+$) and negative turnout bias ($B^-$) are:
> 
> $ B^+(aye, nay) = aye \times b > nay \ B^-(aye, nay) = aye > nay \times b \ b \equiv \sqrt{t / T} $
> 
> Where $t$ is the total number of voting tokens and $T$ is the total number of tokens issued in the system. $aye$ is the number of aye votes, and $nay$ the number of nay votes.

But its not very pretty...

Suggest replacing with an image of the equation, AQB diagrams, and the governance mechanism diagram like the one that Dr Gavin Wood presented at EdCon Toronto https://youtu.be/VsZuDJMmVPY?t=24732

### LINKED ISSUE #155: Add decision graphs to governance page
Add decision graphs comparing majority-carries, static quorum, and super majority.

### PR DESCRIPTION
**Pull request by lsaether**
fixes #151 fixes #155 

starts #12 and #158 

- Fixes rendering of Latex in markdown
- Adds diagrams to governance page
- Adds decision graphs 
- Starts council overview
- Removes duplicated content

### CODE REVIEW

File: docs/polkadot/node/governance/index.md [hunk 1]
```diff
@@ -51,33 +48,43 @@ If the referenda only has 20% turnout, the tally of "aye" votes has to reach 14%
...
-* **Bonding** - 
-New parachains are added by bonding tokens. Outdated or non-useful parachains are removed by removing bonded tokens. This is a form of Proof of Stake.
+## [Usage of DOT](../../learn/DOT.md#dots-for-governance)
 
 ## Guides
 
-### [How to create a proposal ?]()
-### [How to join the council ?]()
-### [How to propose a referendum ?]()
+### [How to create a proposal]()
```
Comment (laboon) [inline docs/polkadot/node/governance/index.md:88]: These links appear to be missing.  If they don't exist, perhaps it's better to just make a comment and add them in later?

File: docs/polkadot/node/governance/index.md [hunk 2]
```diff
@@ -51,33 +48,43 @@ If the referenda only has 20% turnout, the tally of "aye" votes has to reach 14%
...
+
+- Making sensible proposals for referenda.
+- Cancelling any uncontroversially dangerous or malicious referenda.
+
+### How does one join the council?
+
+An account wishing to join the council must first signal their candidacy and submit the required bond. The requisite bond for council membership is configurable by the governance process. The voting process is continuous and voters may at any time register approval of an account for inclusion to the council. Each voting period, the tally is taken for new votes and the winners of the vote are allowed to present themselves as active for the council seat. The council seat is then active for duration of the term. The number of desired seats for the council is a parameter that can be determined by the governance process and is envisioned to be 24 for Polkadot with 12 month terms.
 
-## [Council](https://github.com/paritytech/polkadot/wiki/Governance#council)
+### How are council motions made? What can they do?
```
Comment (laboon) [inline docs/polkadot/node/governance/index.md]: There's no content in this section...

File: docs/polkadot/node/governance/index.md [hunk 3]
```diff
@@ -51,33 +48,43 @@ If the referenda only has 20% turnout, the tally of "aye" votes has to reach 14%
...
 
-Since not everyone is interested in participating in governance, there is a council entity to represent the passive token holders. Election of council members is done through an approval vote.
+### How does one vote for a council member?
 
-The council is called upon primarily for two tasks of governance: proposing sensible referenda, and cancelling uncontroversially dangerous or malicious referenda.
+Voters must also submit a bond to signal approvals for council seats. If a voter becomes inactive, they may be "reaped" by an external actor and their bond claimed as a reward. The process of the running votes is known as [approval voting](https://en.wikipedia.org/wiki/Approval_voting).
```
Comment (laboon) [inline docs/polkadot/node/governance/index.md:70]: It seems like this is missing some content, as it starts with "Voters must also..."

Or should it just start with "Voters must submit a bond.."?

File: docs/polkadot/learn/governance.md [hunk 4]
```diff
@@ -18,6 +18,36 @@ Despite this, by taking advantage of the fact that turnout is rarely 100%, we ca
 
 Another mechanism used, one again which favors the *nay* side (or the *status quo*), is to require a super-majority approval. This works on two principles: firstly that the status quo tends to be safer than any change, and thus should have some bias towards it. Secondly that, like all means of empirical measurement, there is inevitably going to be some degree of inaccuracy and volatility over time. A result could be 51%-49% one month and then change to 49%-51% the following month because of macro-environmental factors such as the economy, social-demographic progression or even the weather. Given the costs involved in enacting the changes of a proposal, it is advantageous to ensure that a result would not likely flip within a short period of time after enactment, and thus a supermajority should approve the non-status-quo option. (As an example of why this can be important, a super-majority was indeed not required in the UK's "Brexit" referendum, resulting in a result that, two years on from the vote but before the enactment, appears indeed to have flipped).
 
+```
```
Comment (laboon) [inline docs/polkadot/learn/governance.md:21]: Just a comment, no action required:

ASCII drawings are OK for now I think, especially for simple things, but do you think that we should add actual images in the future?

**Review by laboon — CHANGES_REQUESTED** [review]
Overall good, a few minor comments/concerns

File: docs/polkadot/learn/governance.md (hunk 4 above)
Comment (lsaether) [reply docs/polkadot/learn/governance.md:21]: ASCII drawings are good enough to show a point and come out better than what I would be able to do in GIMP. Totally real images would be better but it depends on the level of priority and the capacity of the people with those skills.

Comment (laboon) [reply docs/polkadot/learn/governance.md:21]: Yeah I think it's low priority for now, just thinking ahead

File: docs/polkadot/node/governance/index.md (hunk 1 above)
Comment (lsaether) [reply docs/polkadot/node/governance/index.md:88]: yes they are TODO

**Comment by ansonla3** [comment]
@laboon I will pick up #12  & #158   :)

**Review by ansonla3 — APPROVED** [review]

**Comment by lsaether** [comment]
PR will follow from @ansonla3 completing the governance content.

**Pull request merged**